warehouse/models/mart/transit_database/bridge_services_x_county_geography.sql

```diff
@@ -0,0 +1,29 @@
+{{ config(materialized='table') }}
+
+WITH services AS ( -- noqa
+    SELECT *
+    FROM {{ ref('int_transit_database__services_dim') }}
+),
+
+geography AS ( -- noqa
+    SELECT *
+    FROM {{ ref('int_transit_database__county_geography_dim') }}
+),
+
+bridge_services_x_county_geography AS (
+    SELECT
+        services.source_record_id AS source_record_id,
```

Comment:
Same comment as orgs, we'd want `services.key AS service_key` -- following: https://github.com/cal-itp/data-infra/blob/main/warehouse/models/mart/transit_database/bridge_organizations_x_ntd_agency_info.sql